camille/source/windiris.py

```diff
@@ -75,8 +75,7 @@ def windiris(root, tzinfo=pytz.utc):
         Function for reading windiris time series from db file
 
         installation : str
-            Installation directory inside root, part of the file name
-            and sql table name at the same time
+            Name of the windmill to collect data from
```

Comment:
Not entirely sure installation necessarily corresponds to a single windmill.

Reply:
Options?
Do you think just "Installation name" would be fine?

Reply:
"Name of the installation to collect data from" would be fine.